rmgpy/data/kinetics/family.py

```diff
@@ -134,6 +134,29 @@ def __reduce__(self):
                                    self.reverse,
                                    ))
 
+    def __repr__(self):
+        """
+        Return a string representation that can be used to reconstruct the
+        object.
+        """
+        string = 'TemplateReaction('
+        if self.index != -1: string += 'index={0:d}, '.format(self.index)
+        if self.label != '': string += 'label={0!r}, '.format(self.label)
+        if self.reactants is not None: string += 'reactants={0!r}, '.format(self.reactants)
+        if self.products is not None: string += 'products={0!r}, '.format(self.products)
+        if self.specificCollider is not None: string += 'specificCollider={0!r}, '.format(self.specificCollider)
+        if self.kinetics is not None: string += 'kinetics={0!r}, '.format(self.kinetics)
+        if not self.reversible: string += 'reversible={0}, '.format(self.reversible)
```

Comment:
My concern with not printing fields that are default values is that when this is used, it isn't obvious whether the object has the default value of a field or the field just wasn't printed and even if they know that the person using it may not know what the default value is suppose to be.